### PR TITLE
Make APCu key prefix configurable

### PR DESCRIPTION
**Pull request by gkralik**
Make the APCu key prefix configurable for the APC storage adapter.

This is useful if the default `prom` prefix is already used.

### CODE REVIEW

File: src/Prometheus/Storage/APC.php [hunk 1]
```diff
@@ -11,21 +11,26 @@
 
 class APC implements Adapter
 {
-    const PROMETHEUS_PREFIX = 'prom';
```
Comment (LKaemmerling) [inline src/Prometheus/Storage/APC.php:14]: Removing this is a breaking change, we should keep the constant here

File: src/Prometheus/Storage/APC.php [hunk 2]
```diff
@@ -11,21 +11,26 @@
...
 
     /**
      * APC constructor.
      *
+     * @param string $prometheusPrefix Prefix for APCu keys (defaults to 'prom').
+     *
      * @throws StorageException
      */
-    public function __construct()
+    public function __construct(string $prometheusPrefix = 'prom')
```
Comment (LKaemmerling) [inline src/Prometheus/Storage/APC.php:24]: I would prefer to reuse the const here:
```suggestion
    public function __construct(string $prometheusPrefix = self::PROMETHEUS_PREFIX)
```

**Review by LKaemmerling — CHANGES_REQUESTED** [review]
Hey @gkralik,

thank you for the MR! Great job. I have a few comments and can you also please add a test case for changing the prefix?

The best place for the test would be here:
https://github.com/PromPHP/prometheus_client_php/blob/master/tests/Test/Prometheus/APC/CollectorRegistryTest.php

**Comment by gkralik** [comment]
I re-introduced the `PROMETHEUS_PREFIX` const and added a test for custom prefix usage.

**Review by LKaemmerling — APPROVED** [review]
Looks good for me 👍🏻 good job thank you!

**Pull request merged**